qir/qsharp-target-tests/TargetTestScripts/__init__.py

```diff
@@ -1,63 +1,509 @@
+    "Decompositions/R1ControlledAdjoint.qs",
+    "Decompositions/R1Frac.qs",
+    "Decompositions/R1FracAdjoint.qs",
+    "Decompositions/R1FracControlled.qs",
+    "Decompositions/R1FracAdjointControlled.qs",
+    "Decompositions/R1FracControlledAdjoint.qs",
+    "Decompositions/RFrac.qs",
+    "Decompositions/RFracAdjoint.qs",
+    # TODO(tfr): not straightline input "Decompositions/RFracControlled.qs",
+    # TODO(tfr): not straightline input "Decompositions/RFracAdjointControlled.qs",
+    # TODO(tfr): not straightline input "Decompositions/RFracControlledAdjoint.qs",
+    "Decompositions/Reset.qs",
+    "Decompositions/ResetAdjoint.qs",
+    "Decompositions/ResetControlled.qs",
+    "Decompositions/ResetAdjointControlled.qs",
+    "Decompositions/ResetControlledAdjoint.qs",
+    "Decompositions/Rx.qs",
+    "Decompositions/RxAdjoint.qs",
+    # TODO(tfr): not straightline input "Decompositions/RxControlled.qs",
+    # TODO(tfr): not straightline input "Decompositions/RxAdjointControlled.qs",
+    # TODO(tfr): not straightline input "Decompositions/RxControlledAdjoint.qs",
+    "Decompositions/Ry.qs",
+    "Decompositions/RyAdjoint.qs",
+    "Decompositions/RyControlled.qs",
+    "Decompositions/RyAdjointControlled.qs",
+    "Decompositions/RyControlledAdjoint.qs",
+    "Decompositions/Rz.qs",
+    "Decompositions/RzAdjoint.qs",
+    "Decompositions/RzControlled.qs",
+    "Decompositions/RzAdjointControlled.qs",
+    "Decompositions/RzControlledAdjoint.qs",
+    "Decompositions/S.qs",
+    "Decompositions/SAdjoint.qs",
+    "Decompositions/SControlled.qs",
+    "Decompositions/SAdjointControlled.qs",
+    "Decompositions/SControlledAdjoint.qs",
+    "Decompositions/SWAP.qs",
+    "Decompositions/SWAPAdjoint.qs",
+    "Decompositions/SWAPControlled.qs",
+    "Decompositions/SWAPAdjointControlled.qs",
+    "Decompositions/SWAPControlledAdjoint.qs",
+    # TODO(tfr): Not supported by hardware "Decompositions/SetToBasisState.qs",
+    "Decompositions/T.qs",
+    "Decompositions/TAdjoint.qs",
+    "Decompositions/TControlled.qs",
+    "Decompositions/TAdjointControlled.qs",
+    "Decompositions/TControlledAdjoint.qs",
+    "Decompositions/X.qs",
+    "Decompositions/XAdjoint.qs",
+    "Decompositions/XControlled.qs",
+    "Decompositions/XAdjointControlled.qs",
+    "Decompositions/XControlledAdjoint.qs",
+    "Decompositions/Y.qs",
+    "Decompositions/YAdjoint.qs",
+    "Decompositions/YControlled.qs",
+    "Decompositions/YAdjointControlled.qs",
+    "Decompositions/YControlledAdjoint.qs",
+    "Decompositions/Z.qs",
+    "Decompositions/ZAdjoint.qs",
+    "Decompositions/ZControlled.qs",
+    "Decompositions/ZAdjointControlled.qs",
+    "Decompositions/ZControlledAdjoint.qs",
+
+
+    "CodeBlocks/QubitArrays1.qs",
+    "CodeBlocks/QubitArrays2.qs",
+    # TODO(tfr): Does not compile "CodeBlocks/Range1.qs",
+    "CodeBlocks/OffsetBySix.qs",
+    # TODO(tfr): Does not compile "CodeBlocks/ConstantCircuitUsingMutables1.qs",
```

Comment:
What does "does not compile" mean; is the Q# compiler giving a nice error stating it is not supported on that target?

Reply:
It can mean either. It is simply scripts that I would have expected to be deployable that for one reason or the other does not make it through Q#

Reply:
Ok, we need to go over these and make a call on what to do with each one - if the error is justified, then I think we should remove them here. We can proceed with this PR without that, but could you please file a GitHub issue to go over this and either remove or enable the test cases?